packages/autoloader/src/class-plugin-locator.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/* HEADER */ // phpcs:ignore
+
+/**
+ * This class scans the WordPress installation to find active plugins.
+ */
+class Plugin_Locator {
+
+	/**
+	 * Checks a given option for plugin paths.
+	 *
+	 * @param string $option_name  The option that we want to check for plugin information.
+	 * @param bool   $site_option  Indicates whether or not we want to check the site option.
+	 *
+	 * @return array $plugin_paths The list of absolute paths we've found.
+	 */
+	public function find_using_option( $option_name, $site_option = false ) {
```

Comment:
`get_site_option` calls `get_network_option`, which calls `get_option` when the site is not a multisite (see [here](https://core.trac.wordpress.org/browser/tags/5.5.1/src/wp-includes/option.php#L1380)). Do you think we should consider removing the `$site_option` parameter and just using `get_site_option`?

Reply:
My preference would be that we remain explicit here since `get_site_option` also calls some filters and can be short-circuited with the expectation that it's a multisite installation. It feels like using `get_site_option` would be masking these details that might be relevant.